utility/service/service.go

```diff
@@ -147,3 +353,12 @@ func (r *relay) GetRelayChain() RelayChain              { return nil }
 func (r *relay) GetGeoZone() GeoZone                    { return nil }
 func (r *relay) GetToken() AAT                          { return nil }
 func (r *relay) GetSignature() string                   { return "" }
+
+func isChainSupported(chainId string, allChains []string) bool {
+	for _, ch := range allChains {
```

Comment:
Optional NIT: We can use `slices.Index` nowadays :)

Reply:
Thank you for the review. Fixed.